docs/architecture/adr-template.md

```diff
@@ -52,6 +46,12 @@
 > how the author proposes to deal with these incompatibilities. ADR submissions
 > without a sufficient backwards compatibility treatise may be rejected outright.
 
+## Alternatives
```

Comment:
note to reviewer: I moved the `Alternatives` section here because it doesn't make sense to have it before the decision. The decision gives the reader understand of the proposal first. Then, after, does it only make sense to state alternatives.